qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -135,8 +152,20 @@ jobs:
         state: pending
     <% end %> # status_reporting
     on_failure:
+      <% if enable_terraform %>
+      try:
+        do:
+          - task: delete-aks-resource-group
+            file: ci/qa-pipelines/tasks/delete-aks-resource-group.yml
+            params:
+              << : *common-task-params
+          - put: pool.kube-hosts
+            params: {remove: pool.kube-hosts}
+      <% end %> # enable_terraform
+      <% if !enable_terraform %>
       put: pool.kube-hosts
       params: {release: pool.kube-hosts}
+      <% end %> # not enable_terraform
```

Comment:
`# enable_terraform`, not `# not enable_terraform`

Reply:
no its  <% if !enable_terraform %>

Reply:
anh u meant # !enable_terraform

Reply:
change requested